app/main.go

```diff
@@ -214,6 +215,13 @@ func loadStorageEngine(ctx context.Context) (storage.Manager, error) {
 	case "datastore":
 		datastoreManager := datastore.New(ctx, &datastore.Options{})
 		return datastoreManager, nil
+	case "boltdb":
+		if isAppEngine() {
+			log.Fatal("BoltDB storage is not supported on AppEngine")
+		}
+		opts := &boltdb.Options{DBPath: os.Getenv("BOLTDB_PATH")}
```

Comment:
Do you want to document this new environment variable?

Reply:
D'oh, that was staged but not committed, done.